servant-multipart-client/servant-multipart-client.cabal

```diff
@@ -0,0 +1,57 @@
+name:               servant-multipart-client
+version:            0.12
+synopsis:           multipart/form-data (e.g file upload) support for servant
```

Comment:
Could you please update synopsises and descriptions to reflect split packages' purpose? I suppose the original one should be retained only in `servant-multipart` package.

The same goes for module descriptions in `.hs` files.

Reply:
done